lib/modules/us_core_guidance/clinicalnotes_sequence.rb

```diff
@@ -2,14 +2,24 @@
 
 module Inferno
   module Sequence
+    class ClinicalNoteAttachment
```

Comment:
I don't know that this class is providing any value. It seems like `@document_attachments` and `report_attachments` could just be hashes.

Reply:
Create an object model so the code can use document_attachments.attachment instead of document_attachments[:attachment]

Reply:
But will change to hash to reduce code size.